json-endpoints/jetpack/class.jetpack-json-api-sync-endpoint.php

```diff
@@ -5,6 +5,25 @@ class Jetpack_JSON_API_Sync_Endpoint extends Jetpack_JSON_API_Endpoint {
 	protected $needed_capabilities = 'manage_options';
 
 	protected function result() {
+		$args = $this->input();
+
+		if ( isset( $args['clear'] ) && $args['clear'] ) {
+			// clear sync queue
+			require_once dirname(__FILE__).'/../../sync/class.jetpack-sync-sender.php';
+
+			$sender = Jetpack_Sync_Sender::getInstance();
+			$sync_queue = $sender->get_sync_queue();
+			$sync_queue->reset();
+		}
+
+		if ( isset( $args['force'] ) && $args['force'] ) {
```

Comment:
Should we clear the queue everytime we force sync items anyways? 
The only thing we might loose by clearing the queue are events such as login or logout etc.